network-api/networkapi/wagtailpages/pagemodels/campaigns.py

```diff
@@ -422,3 +439,13 @@ def get_context(self, request):
     class Meta:
         verbose_name = "Banner Page"
         verbose_name_plural = "Banner pages"
+
+    def clean(self):
+        if self.narrowed_page_content and self.aside_page_content:
+            ERROR_MESSAGE = "Aside AND Narrow formats cannot be used together. "
+            raise ValidationError({
+                'narrowed_page_content': ValidationError(ERROR_MESSAGE),
+                'aside_page_content': ValidationError(ERROR_MESSAGE)
+            })
+
+        return super().clean()
```

Comment:
What do you think about adding small tests for this custom validation logic?

Reply:
Good point, this has now been removed since the rework though